### ISSUE TITLE
Instruct Template

### ISSUE THREAD

**Issue by riccardomusmeci** [apple/corenet#24]
Hi there, looking at OpenELM instruct template what I understood is that the template is something like this:

<|system|>
You are a helpful AI assistant.</s>
<|user|>
How to explain Internet for a medieval knight?</s>
<|assistant|>

Is that right? With MLX the model is not properly working on instruct mode.

**Comment by louis845** [comment]
Yes, this would be really useful. I am trying to use the OpenELM instruct models but I can't find out what the instruct template is.

**Comment by purefall** [comment]
Is there any update regarding the chat template and tokenizer used for instruction tuning?

Especially when using OpenELM 1.1B Instruct models from Huggingface, I only get "<<<<<<" as the output. 

I used tokenizer of "meta-llama/Llama-2-7b-hf" and original template from Llama2, but the output is garbage.... It would be great if you can clarify

**Comment by SongTonyLi** [comment]
I'm also confused here. If indeed we need to use meta-llama/Llama-2-7b-hf [chat template](https://discuss.huggingface.co/t/issue-with-llama-2-chat-template-and-out-of-date-documentation/61645/3), so why does the source code also included another different [chat template](https://github.com/apple/corenet/blob/2261885b6696950aaf481a862e8926921ef1a067/projects/openelm/instruction_tuning/openelm-instruct.yaml#L7).
